test/library/packages/URL/RunServer.chpl

```diff
@@ -0,0 +1,64 @@
+use Subprocess;
+use OS.POSIX;
+use URL;
+use Time;
+use FileSystem;
+
+config const host = "127.0.0.1";
+config const port = "8000";
```

Comment:
Could you move these into `regionArgReader.chpl` and then pass them into `startServer` as formals. I think this would make the test itself more readable, since it would be clearer where the `host` and `port` variables come from.